tests/functional/pyocf/types/volume_core.py

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+from .core import Core
+from .volume_exp_obj import ExpObjVolume
+from pyocf.types.io import IoDir
+
+
+class CoreVolume(ExpObjVolume):
+    def __init__(self, core, uuid=None):
+        super().__init__(core, uuid)
+        self.core = core
+
+    def md5(self):
+        return self.core.exp_obj_md5()
```

Comment:
Don't we want to move this functionality from `Core` to `CoreVolume`? `Core` is supposed to represent management object, while `CoreVolume` is I/O interface. Calculating MD5 sounds more like a feature of the latter.

Reply:
Ok, moved all I/O methods to volumes